config/foreman-proxy-content.yaml

```diff
@@ -14,6 +14,7 @@
 :hiera_config: ./config/foreman-hiera.yaml
 :facts:
   tuning: 'default'
+  mongo_cache_size: 1
```

Comment:
Technically this isn't really needed. `get_custom_fact` should return `nil`. `nil != something` so that should store the fact. This is also the upgrade scenario now since there's no migration. The reason we set one for tuning is that there the value is read for the `--tuning` parameter and we need to show something.

Reply:
good to know this, I was uncomfortable putting a random default value `1` here.